### DISCUSSION TITLE
Maintainers?

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/defaang#297] [General]
@iShibi @mark-omarov @rohitdasu - I noticed you haven't been too active here recently.

Did you still want to be maintainers here?

I tried to message you on Discord, but I couldn't find all of you there.

**Comment by iShibi** [comment]
I'll step down.

**Reply by rohitdasu** [reply]
Me too

**Comment by subhoghoshX** [comment]
Even though I'm not tagged here, I'm sorry for not being more active. I'll make an effort to work on my schedule.

**Comment by mark-omarov** [comment]
Hey YK, I'm busy parenting, working, can't work on it rn, sorry. Maybe will get more time later.
